tests/plotting/test_plotting.py

```diff
@@ -1646,33 +1646,44 @@ def test_image_properties():
     p.close()
 
 
-def test_volume_rendering():
-    # Really just making sure no errors are thrown
-    vol = examples.load_uniform()
-    vol.plot(volume=True, opacity='linear')
+def test_volume_rendering_from_helper(uniform):
+    uniform.plot(volume=True, opacity='linear', before_close_callback=verify_cache_image)
```

Comment:
We should really just make a pytest fixture context that sets `verify_cache_image` as the default `before_close_callback`... to do in a later PR though